devtools/platform-descriptor-json/src/main/resources/codestarts/funqy-http-example/java/src/test/java/org/acme/funqy/FunqyTest.java

```diff
@@ -0,0 +1,66 @@
+package org.acme.funqy;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.equalTo;
+
+@QuarkusTest
+public class FunqyTest {
+
+    @Test
+    public void testHello() {
+        given()
+                .given().get("/hello")
```

Comment:
I am not sure of how this work for `funqy` but don't we need to namespace those path too?